test/fixtures/books.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+# This model initially had no columns defined. If you add columns to the
+# model remove the "{}" from the fixture names and add the columns immediately
+# below each fixture, per the syntax in the comments below
+#
+one: {}
```

Comment:
I'm not too familiar with Ruby testing but I think this is used to create mock test data? https://thoughtbot.com/blog/we-need-to-talk-about-fixtures